Module2/springhometask/src/main/java/com/epam/spring/domain/UserAccountEntity.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.spring.domain;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class UserAccountEntity {
```

Comment:
Why you need UserAccountEntity class? Seems UserEntity could be used.

Reply:
I've created it, because it is needed by the conditions of the task.
`Add new entity to the application - UserAccount, it should store the amount of prepaid money user has in the system, which should be used during booking procedure`